src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -312,14 +329,13 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
         int theStart = start;
         int theEnd = end;
 
-        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start - 1, end - start + 1);
         //make sure that referenceAllele matches reference
-        final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElseThrow(() -> new RuntimeException("How did that happen? No reference allele?"));
-
-        if (!refString.equalsIgnoreCase(refAllele.getBaseString())) {
+        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start - 1, end - start + 1);
```

Comment:
seems like code duplication, could you move this line into `referenceAlleleMatchesReference`?